src/bindings/python/flux/core/handle.py

```diff
@@ -247,8 +247,11 @@ def msg_watcher_create(
         topic_glob="*",
         args=None,
         match_tag=raw.FLUX_MATCHTAG_NONE,
+        rolemask=None,
```

Comment:
would it be worthwhile to set this to FLUX_ROLE_OWNER?  so it's obvious what the default is?  It wouldn't match the C code, but C is different than Python.  ... ehhh

Reply:
It seemed better to have the default behavior preserved rather than forcing a flag here 🤷